packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -0,0 +1,330 @@
+import { Chain } from 'wagmi'
+import { useDebounce } from 'react-use'
+import { ChevronLeftIcon, XMarkIcon } from '@heroicons/react/24/outline'
+
+import { chainInfo, getSupportedNetworks, isNetwork } from '../../util/networks'
+import { useAccountType } from '../../hooks/useAccountType'
+import { useIsTestnetMode } from '../../hooks/useIsTestnetMode'
+import { SearchPanel } from './SearchPanel/SearchPanel'
+import { SearchPanelTable } from './SearchPanel/SearchPanelTable'
+import { twMerge } from 'tailwind-merge'
+import { TestnetToggle } from './TestnetToggle'
+import { useArbQueryParams } from '../../hooks/useArbQueryParams'
+import {
+  PanelWrapperClassnames,
+  onPopoverButtonClick,
+  onPopoverClose
+} from './SearchPanel/SearchPanelUtils'
+import { getBridgeUiConfigForChain } from '../../util/bridgeUiConfig'
+import { getWagmiChain } from '../../util/wagmi/getWagmiChain'
+import { useNetworks } from '../../hooks/useNetworks'
+
+type NetworkInfo = {
+  chainId: number
+  type: 'core' | 'orbit'
+}
+
+const chainGroupInfo = {
+  core: {
+    name: 'CORE CHAINS',
+    description: 'Chains managed directly by Ethereum or Arbitrum'
+  },
+  orbit: {
+    name: 'ORBIT CHAINS',
+    description: 'Independent chains supported by Arbitrum technology'
+  }
+}
+
+function NetworkRow({
+  networkInfo,
+  style,
+  onClick,
+  close
+}: {
+  networkInfo: NetworkInfo
+  style: CSSProperties
+  onClick: (value: Chain) => void
+  close: (focusableElement?: HTMLElement) => void
+}) {
+  const { chainId } = networkInfo
+  const { network } = getBridgeUiConfigForChain(chainId)
+  const chain = getWagmiChain(chainId)
+
+  const handleClick = () => {
+    onClick(chain)
+    close() // close the popover after option-click
+  }
+
+  return (
+    <button
+      onClick={handleClick}
+      onKeyDown={e => {
+        if (e.key === 'Enter') {
+          handleClick()
+        }
+      }}
+      key={chainId}
+      style={style}
+      type="button"
+      aria-label={`Switch to ${network.name}`}
+      className={twMerge(
+        'flex h-[90px] w-full items-center gap-4 px-6 py-2 text-lg hover:bg-black/10'
+      )}
+    >
+      <span className="flex h-6 w-6 shrink-0 items-center justify-center lg:h-6 lg:w-6">
+        <Image
+          src={network.logo}
+          alt={`${network.name} logo`}
+          className="h-full w-auto"
+          width={24}
+          height={24}
+        />
+      </span>
+      <div className={twMerge('flex flex-col items-start gap-1')}>
+        <span className="truncate leading-none">{network.name}</span>
+        {chainInfo[chainId] && (
+          <>
+            <p className="whitespace-pre-wrap text-left text-xs leading-[1.15]">
+              {chainInfo[chainId]!.description}
+            </p>
+            <p className="text-[10px] leading-none">
+              {chainInfo[chainId]!.chainType} Chain,{' '}
+              {chainInfo[chainId]!.nativeCurrency} is the native gas token
```

Comment:
can get it with `getBridgeUiConfigForChain(chainId).nativeTokenData?.symbol ?? 'ETH'` but I also left a comment for `chainInfo` in general